docs/sources/next/javascript-api/k6-experimental/redis/client/client-decr.md

```diff
@@ -26,15 +26,8 @@ Decrements the number stored at `key` by one. If the key does not exist, it is s
 ```javascript
 import redis from 'k6/experimental/redis';
 
-// Get the redis instance(s) address and password from the environment
-const redis_addrs = __ENV.REDIS_ADDRS || '';
-const redis_password = __ENV.REDIS_PASSWORD || '';
-
 // Instantiate a new redis client
-const redisClient = new redis.Client({
-  addrs: redis_addrs.split(',') || new Array('localhost:6379'), // in the form of 'host:port', separated by commas
-  password: redis_password,
-});
+const redisClient = new redis.Client('redis://localhost:6379');
```

Comment:
This not only updates to the new API, but also removes using a password in the example.

Is this intentional? Do we still have examples with password?

Reply:
That's a fair point! We do illustrate using the password in the `Client` documentation, though 👍🏻